sdk/communication/azure-communication-identity/azure/communication/identity/_shared/utils.py

```diff
@@ -17,6 +16,14 @@
 from azure.core.credentials import AccessToken
 
 def _convert_datetime_to_utc_int(expires_on):
+    """
+    Converts DateTime in local time to the Epoch in UTC in second.
+
+    :param input_datetime: Input datetime
```

Comment:
you have to change the param name to `input_datetime` , same for other duplications in other packages

Reply:
well spotted! :)
that was a copy-paste from chat. it seems that the linter doesn't check whether comments are matching param names.
fixed in 191dc62